dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/file/LogFileWriter.kt

```diff
@@ -105,6 +108,19 @@ internal class LogFileWriter(
         return jsonLog.toString()
     }
 
+    private fun addLogUserInfo(log: Log, jsonLog: JsonObject) {
```

Comment:
you will have to move this part into the new LogSerializer. I think this is the only conflict you need to solve.